### PR TITLE
DDL instructional videos link added

### PR DESCRIPTION
**Pull request by abubaker47**
DDL instructional video for English, Farsi, and Pashto language added, Story weaver sign-in link added on the banner.

### CODE REVIEW

**Review by uyscuti-wiki — CHANGES_REQUESTED** [review]
Thanks Faisal. Looks pretty good, few comments inline and one in Jira.

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -16,7 +16,12 @@
     @include('layouts.messages')
     <div class="ddl-top-news">
         <div>
-            <p>@lang('Watch a video to learn more about our work in Afghanistan'): <a href="https://www.youtube.com/watch?v=bF5dpED9W64" target="_blank" title="Our work in Afghanistan">@lang('Learn More')</a></p>
+            <p>@lang("Access children's storybooks through Storyweaver, click here"): 
+                <a href="{{ route('storyweaver-confirm') }}" target="_blank" title="@lang('StoryWeaver Library')">
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/home.blade.php]: Added the $landing_page parameter.

File: resources/views/layouts/banner.blade.php [hunk 2]
```diff
@@ -97,7 +97,7 @@
             </li>
             @if ($loop->index == 1) {{-- where 0 is Home, 1 is DDL Library. We want it next to DDL Library. --}}
                 <li>
-                    <a href="{{ route('storyweaver-confirm') }}" title="StoryWeaver"><img src="{{ URL::to(config('constants.ddlmain_s3_file_storage_url').'/public/img/storyweaver-logo.svg') }}" class="storyweaver-logo"> @lang('StoryWeaver Library')</a>
+                    <a href="{{ route('storyweaver-confirm') }}" title="@lang('StoryWeaver Library')"><img src="{{ URL::to(config('constants.ddlmain_s3_file_storage_url').'/public/img/storyweaver-logo.svg') }}" class="storyweaver-logo"> @lang('StoryWeaver Library')</a>
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/views/layouts/banner.blade.php]: Added the `$landing_page` parameter.

File: resources/lang/fa.json [hunk 3]
```diff
@@ -1,5 +1,8 @@
 {
     "Darakht-e Danesh Online Library": "کتابخانه درخت دانش",
+    "How to Use the Darakht-e Danesh Library": "با کتاب‌خانه دیجیتالی درخت دانش و امکانات آن بیشتر آشنا شوید",
+    "StoryWeaver Library": "کتاب‌خانه ستوری ویوور",
+    "Access children's storybooks through Storyweaver, click here": "برای دسترسی به کتاب‌های داستان برای کودکان نشر شده توسط ستوری ویوور، اینجا را کلیک کنید",
```
[the remark this answers is not in the document]
Comment (abubaker47) [reply resources/lang/fa.json]: Agree, removed the translations

**Pull request merged**